### ISSUE TITLE
Dozer for Chrome Request

### ISSUE THREAD

**Issue by qplus7** [Mortennn/Dozer#182]
Hello - thanks for building this. The amount of icons on my system tray had been driving me insane until I found this, today! 

Is it possible to build something like this for Chrome extensions? I have tons of extensions that I use but would love to hide them if needed.

Thanks in advance!

**Comment by Parakeet709** [comment]
I don't think that extensions can change the current pinned extensions in Chrome at all. I think it would require a Chromium browser with modified code to make this possible.
It might be possible in normal Chrome, but I am not a developer, so I don't know if it is possible.
